django-backend/fecfiler/transactions/migrations/0022_add_debt_foreign_key.py

```diff
@@ -17,7 +17,7 @@ class Migration(migrations.Migration):
                 blank=True,
                 null=True,
                 on_delete=django.db.models.deletion.CASCADE,
-                related_name="debt_repayments",
```

Comment:
i think this might cause problems on DEV since this migration already ran on it

Reply:
You'll need to revert this change and add the related_name change to a later migration

Reply:
Actually! according to SO, related_name is not state on the DB so it doesn't matter where it shows up in the migration https://stackoverflow.com/a/19815633